cnd/Cargo.toml

```diff
@@ -34,11 +31,10 @@ http-api-problem = "0.13"
 hyper = "0.12"
 lazy_static = "1"
 libp2p = { version = "0.13" }
-libp2p-core = { version = "0.13" }
```

Comment:
How come it's here? don't we have tomlfmt running in the CI?

Reply:
Should actually, not know why it slipped through :man_shrugging:

Reply:
I think you fixed it, for tomlfmt you have to explicitly pass each cargo toml file.